src/constants.ts

```diff
@@ -0,0 +1,4 @@
+import { getNumbers } from './utils';
```

Comment:
It seems like the `getNumbers` function is being imported but not used within this file. If this import is not necessary, consider removing it to keep the codebase clean and to avoid unnecessary dependencies.

Reply:
It is used and it is need in both files.